cloud/src/meta-service/meta_service.cpp

```diff
@@ -1811,14 +1814,9 @@ static bool check_delete_bitmap_lock(MetaServiceCode& code, std::string& msg, st
     }
     return true;
 }
-static bool use_new_version_random() {
-    std::mt19937 gen {std::random_device {}()};
-    auto p = 0.5;
-    std::bernoulli_distribution inject_fault {p};
-    if (inject_fault(gen)) {
-        return true;
-    }
-    return false;
+
+void MetaServiceImpl::get_version(std::string& use_version, std::string& instance_id) {
```

Comment:
use a specific name like get_delete_bitmap_lock_version,
get_versionb is too generous